lib/widgets/expandablelist.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:mixeditems/repository/repo.dart';
+
+import '../models/listitem.dart';
+
+class ExpandableListSection implements ListItem {
+  final String title;
+
+  ExpandableListSection(this.title);
+
+  //Expandable List starts.
+
+  @override
+  Widget buildBody(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        ExpansionTile(
+          title: Text(title),
+          controlAffinity: ListTileControlAffinity.leading,
+          children: <Widget>[
+            ListTile(title: Text('This is tile number 1')),
+            ListTile(title: Text('This is tile number 2')),
+            ListTile(title: Text('This is tile number 3')),
+            ListTile(title: Text('This is tile number 4')),
+            ListTile(title: Text('This is tile number 5')),
+          ],
+        ),
+        ExpansionTile(
```

Comment:
When expanded this tile has double its content's height with half of it being blank

Reply:
I will remove the nested scrolling.